projects/plugins/jetpack/tests/php/_inc/lib/test-class-jetpack-podcast-helper.php

```diff
@@ -69,12 +69,14 @@ public function test_get_track_data_find_episode() {
 			->method( 'load_feed' )
 			->will( $this->returnValue( $rss ) );
 
+		$id = wp_unique_id( 'podcast-track-' );
```

Comment:
I was a bit surprised to see this in here, but  ¯\_(ツ)_/¯

Reply:
This was needed because I refactored the `add_admin_menu_separator` function to use `wp_unique_id` as well, so that affected this test that was using the same function and expecting the numeric ID that has been taken now by `add_admin_menu_separator`.